docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -176,33 +111,160 @@ This example configuration file accepts input from an OpenTelemetry Agent, proce
+You can also enable {ot-extension}[`extensions`] for tasks like monitoring the health of the collector.
+
+TIP: We recommend using {metricbeat-ref}/metricbeat-overview.html[{metricbeat}] to get a mature collector with more integrations
+and integrated visualizations to collect infrastructure metrics.
+
+[float]
+[[open-telemetry-elastic-config]]
+===== Elastic exporter configuration options
+
+|=== 
+
+| `apm_server_url` | Elastic APM Server URL. (required).
+
+| `api_key` | Credential for {apm-server-ref-v}/api-key.html[API key authorization]. Must also be enabled in Elastic APM Server. (optional)
+
+| `secret_token` | Credential for {apm-server-ref-v}/secret-token.html[secret token authorization]. Must also be enabled in Elastic APM Server. (optional)
+
+| `ca_file` | Root Certificate Authority (CA) certificate for verifying the server's identity if TLS is enabled. (optional)
+
+| `cert_file` | Client TLS certificate. (optional)
+
+| `key_file` | Client TLS key. (optional)
+
+| `insecure` | Disable verification of the server's identity if TLS is enabled. (optional)
+
+|===
+
+[float]
+[[instrument-apps-collector]]
+===== Instrument applications
+
+To export traces and metrics to the OpenTelemetry Collector, ensure that you have instrumented your services and applications
+with the OpenTelemetry API, SDK, or both. For example, if you are a Java developer, you need to execute your Java app using the
+https://github.com/open-telemetry/opentelemetry-java-instrumentation[OpenTelemetry agent for Java]. 
+
+By defining the following environment variables, you can customize the OTLP endpoint the agent will use to communicate with
+APM Server.
+
+[source,bash]
+----
+export OTEL_RESOURCE_ATTRIBUTES=service.name=frontend,service.version=1.0-SNAPSHOT,deployment.environment=staging
+export OTEL_EXPORTER_OTLP_ENDPOINT=https://apm_server_url:8200
+java -javaagent:/path/to/opentelemetry-javaagent-all.jar \
+     -jar target/frontend-1.1.jar
+----
+
+|=== 
+
+| `OTEL_RESOURCE_ATTRIBUTES` | The service name to identify your application.
+
+| `OTEL_EXPORTER_OTLP_ENDPOINT` | APM Server URL. The host and port that APM Server listens for events on.
+
+|===
+
+You are now ready to collect <<open-telemetry-elastic-traces-metrics,traces and metrics>>, <<open-telemetry-elastic-verify,verifying metrics>>,
+and <<open-telemetry-elastic-kibana,visualizing metrics>> in {kib}.
 
-NOTE: For more information about getting started with an OpenTelemetry Collector,
-see the {ot-collector}[OpenTelemetry collector] docs.
+[[open-telemetry-elastic-protocol]]
+==== APM Server native support of OpenTelemetry protocol
+
+experimental::[This feature is experimental and may be changed in a future release. It is only available in a self-managed environment.]
+
+The APM Server native support of OpenTelemetry protocol integration allows you to send collected telemetry data directly from your applications to APM Server.
```

Comment:
Maybe one of these instead?

> The APM Server native support of the OpenTelemetry protocol allows you to send collected telemetry data directly from your applications to APM Server.

> The APM Server native OpenTelemetry protocol integration allows you to send collected telemetry data directly from your applications to APM Server.

Reply:
Good suggestions. I went with the first one.